Automattic/wp-calypso
# Framework: CSS: Split the vendor bundle out of the main CSS file

This pull request moves the vendor CSS to a separate file, so we can split the CSS into different files and load it conditionally.
  
#### Testing instructions
  
1. Run `git checkout try/move-vendor-css` and start your server
2. Open the [`Home` page](http://calypso.localhost:3000/)
3. Check that everything looks the same as before
 
#### Notes
Do we need to do anything at a systems level minify this file?

#### Reviews
